client/js/screens/BeginChallenge/styles.js

```diff
@@ -2,7 +2,61 @@ import { StyleSheet } from "react-native";
 import { colors, fonts } from "../../config/styles";
 
 const styles = StyleSheet.create({
- 
+  center: {
+    alignItems: "center",
+    flex: 1.25,
+    justifyContent: "space-around",
+    paddingTop: 120
```

Comment:
Use flexbox properties. Should not do paddingTop:120.

Reply:
Resolved